travis/build.sh

```diff
@@ -2,7 +2,7 @@
 
 set -e
 
-pytest
+python -m pytest
```

Comment:
`pytest` doesn't run 3 of 4 tests in directory `tests`, but `python -m pytest` runs them all.